src/actions/clientActions.js

```diff
@@ -45,7 +45,8 @@ export const searchClients = (query, type) => dispatch => {
         }
       })
       .then(res => {
-        dispatch({ type: FARMER_SEARCH_SUCCESS, payload: res.data });
+        dispatch({ type: FARMER_SEARCH_SUCCESS, payload: res.data, headers:res.headers});
```

Comment:
For some reason I didn't think you could add things besides type and payload, I like this approach much better than my initial attempt with organizations